src/js/index/todolist.js

```diff
@@ -0,0 +1,121 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $todoList = document.querySelector("#todo-list");
+const $todoCount = document.querySelector("strong");
+const $filters = document.querySelector(".filters");
+
+$todoInput.addEventListener("keyup", onAddTodoItem);
+$todoList.addEventListener("click", onToggleTodoItem);
+$todoList.addEventListener("dblclick", onEditTodoItem);
+$filters.addEventListener("click", onFilter);
+
+function onAddTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
+    if (event.key === "Enter" && todoTitle !== "") {
+        todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
+        updateTodoCount();
+        event.target.value = "";
+        changeState();
+    }
+}
+
+function onToggleTodoItem(event) {
+    if (event.target.classList.contains("toggle")) {
+        event.target.closest("li").classList.toggle("completed");
+        event.target.closest("input").toggleAttribute("checked");
+        changeState()
+    } else if (event.target.classList.contains("destroy")) {
+        event.target.closest("li").remove();
+        updateTodoCount();
+        changeState();
+    }
```

Comment:
```suggestion
    if (event.target.classList.contains("toggle")) {
        event.target.closest("li").classList.toggle("completed");
        event.target.closest("input").toggleAttribute("checked");
        changeState()
        return;
    } 
    
    if (event.target.classList.contains("destroy")) {
        event.target.closest("li").remove();
        updateTodoCount();
        changeState();
        return;
    }
```

else if 는 잘 안쓰는것 같아요(else if 를 쓰면 앞의 if문도 읽어야해서 그런거 같아요)